Storage/base.h

```diff
@@ -184,13 +190,13 @@ struct MutationJournal {
   }
 
   void Clear() {
-    meta_fields.clear();
+    transaction_meta.fields.clear();
     commit_log.clear();
     rollback_log.clear();
```

Comment:
Perhaps nullify `begin_us` and `end_us` in `Clear()` and test it in `AssertEmpty()` as well?